src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -531,6 +534,17 @@ public static String calcMD5(final byte[] bytes) {
     public static String calculateFileMD5( final File file ) throws IOException{
         return Utils.calcMD5(FileUtils.readFileToByteArray(file));
```

Comment:
seems like if we can we should make all the old methods delegate to the new ones

Reply:
Agreed, done.